src/module-1/styles/css/style.css

```
@@ -0,0 +1,156 @@
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+}
+
+body {
+  box-sizing: border-box;
+  font-family: "Montserrat", Helvetica, sans-serif;
+}
+
+button, input[type=submit], input[type=reset] {
+  border: none;
+  font: inherit;
+  cursor: pointer;
+  outline: inherit;
+}
+
+.product {
+  align-items: center;
+  background-color: white;
+  border-radius: 8px;
+  box-shadow: 0 2px 8px #00000022;
+  display: flex;
+  flex-direction: column;
+  min-height: 486px;
+  width: 400px;
+  margin: 30px;
+}
+.product__img {
+  height: 272px;
+  margin-top: 31px;
+  max-width: 100%;
+  padding: 15px;
+}
+.product__price {
+  align-self: flex-end;
+  margin-right: 20.49px;
+  margin-left: 247px;
+  margin-top: 36px;
+  color: #2c2c2c;
+  font-size: 18px;
+  font-weight: 400;
+}
+.product__title {
+  align-self: flex-start;
+  line-height: 19px;
+  margin-left: 23.72px;
+  margin-top: 25px;
+  min-height: 19px;
+  color: #2c2c2c;
+  font-size: 16px;
+  font-weight: 400;
+}
+.product__description {
+  line-height: 14px;
+  margin-right: 38px;
+  margin-top: 3px;
+  min-height: 14px;
+  width: 315px;
+  color: #2c2c2c;
+  font-size: 12px;
+  font-weight: 300;
+}
+.product__btn {
+  align-items: flex-start;
+  display: flex;
+  justify-content: center;
+  min-width: 200px;
+  margin-top: 22px;
+}
+.product__btn--heart {
+  background: url("../../img/icons/heart-1@2x.svg") no-repeat;
```

```suggestion
  background: url("../../img/icons/heart.svg") no-repeat;
```  
consider using simplified names for your local images